developers/prerequisites/osgids.md

```diff
@@ -74,6 +74,8 @@ In this example our component needs a [LogService](https://osgi.org/javadoc/r4v4
 
 This is the XML component description. This file is located in the *OSGI-INF* folder and is named *consumer.xml*:
 
+    Note! Please keep in mind that the Service Component Description might be automatically generated using SCR Annotations like @Component, @Reference, @Activate, @Deacticate and etc that are supported e.g. in Apache Felix. Currently these annotations are not used in the openHAB project, so please ignore examples where they are used.
```

Comment:
The hint is nto displayed well in the rendered markdown. Can you please take a look?